### PR TITLE
Feature/lib4

### PR DESCRIPTION
**Pull request by integricho**
This PR adjusts the configuration to comply with latest requirements in librarian 4.0 by adjusting the list of used packages, fixing the collectstatic command (which emptied the static folder under `librarian/static/`), and brings up to date the generation of `librarian.ini`.

### CODE REVIEW

File: rxos/local/librarian-config/src/librarian.ini [hunk 1]
```diff
@@ -10,7 +10,7 @@ port = %PORT%
 
 default_route = %DEFROUTE%
 default_route_args =
-%DEFROUTEARGS%
+    %DEFROUTEARGS%
```
Comment (foxbunny) [inline rxos/local/librarian-config/src/librarian.ini]: Are you sure this is needed? I think the script that handles conversion of list args will add indentation itself.

Comment (integricho) [inline rxos/local/librarian-config/src/librarian.ini]: True, but the menu option was added as just a string, not a list so no indentation fixes are applied.

Comment (foxbunny) [inline rxos/local/librarian-config/src/librarian.ini]: I think the function/script has to be invoked at patch time. I'll recheck.

Comment (foxbunny) [inline rxos/local/librarian-config/src/librarian.ini]: Yeah ok, disregard. It's a single value so it doesn't need to be dedented and listified.

**Pull request merged**